packages/packages.go

```diff
@@ -297,11 +297,63 @@ type Filter struct {
 	Capabilities   []string
 	SpecMin        *semver.Version
 	SpecMax        *semver.Version
+	Discovery      *DiscoveryFilter
 
 	// Deprecated, release tags to be removed.
 	Experimental bool
 }
 
+type DiscoveryFilter struct {
+	Fields DiscoveryFilterFields
+}
```

Comment:
Would it be ok to set those two structs `DiscoveryFilter` and `DiscoveryFilterFields` as private ?